src/redux/services/helm.ts

```diff
@@ -28,6 +29,15 @@ export function isHelmChartFolder(files: string[]) {
   return files.indexOf('Chart.yaml') !== -1 && files.indexOf('values.yaml') !== -1;
 }
 
+/**
+ * check if the k8sResource is supported
+ * @param resource
+ * @returns @boolean
+ */
+function isSupportedHelmResource(resource: K8sResource): boolean {
+  return !JSON.stringify(resource.content).includes('{"{');
```

Comment:
you can use resource.text instead - which won't require you to stringify. Also perhaps we should use a regex to check for a matching closing pair? i.e. '{{.*}}'

Reply:
resolved